slick-testkit/src/test/resources/logback.xml

```diff
@@ -0,0 +1,11 @@
+<configuration debug="true">
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d %-5level [%thread] %logger{0}: %msg%n</pattern>
+        </encoder>
+    </appender>
+    <root level="WARN">
+        <appender-ref ref="STDOUT" />
+    </root>
+    <logger name="slick.jdbc.hikaricp" level="DEBUG"/>
```

Comment:
Do we need this file? This should probably go into `common-test-resources/logback.xml`